client/internal/config.go

```diff
@@ -121,7 +124,15 @@ func createNewConfig(input ConfigInput) (*Config, error) {
 		config.AdminURL = newURL
 	}
 
-	config.IFaceBlackList = defaultInterfaceBlacklist
+	if input.WgIface != "" {
```

Comment:
We need to add similar checks to the ReadConfig function so updates are possible in the up command.

Reply:
updated the code, is this what you meant with modifying readConfig?

Reply:
yes, you just missed the port check.

You can add something like this:
```Go
        if input.WgPort != 0 && input.WgPort !=  config.WgPort {
		config.WgPort =  input.WgPort 
		refresh = true
	}
	
	// previous version config upgrade check
	if config.WgPort == 0 {
		config.WgPort = iface.DefaultWgPort
		refresh = true
	}
```